Kitodo/src/main/java/org/kitodo/production/forms/UserForm.java

```diff
@@ -130,8 +134,15 @@ public String newUser() {
      * @return page or empty String
      */
     public String save() {
-        String login = this.userObject.getLogin();
+        Set<ConstraintViolation<KitodoPassword>> passwordViolations = getPasswordViolations();
+        if(passwordViolations.isEmpty()) {
```

Comment:
```suggestion
        if (!passwordViolations.isEmpty()) {
```
I think this should be negated.